docs/source/use_with_pytorch.mdx

```diff
@@ -200,7 +200,10 @@ You must use a `BatchSampler` if you want the transform to be given full batches
 
 ### Stream data
 
-Loading a dataset in streaming mode allows one to iterate over the dataset without downloading it on disk.
+Stream a dataset by loading it as an [`IterableDataset`]. This allows you to progressively iterate over a dataset without downloading it on disk as well as any local dataset files you have stored.
```

Comment:
```suggestion
Stream a dataset by loading it as an [`IterableDataset`]. This allows you to progressively iterate over a remote dataset without downloading it on disk as well as any local dataset files you have stored.
```

not sure I understand the meaning of the second part of second sentence ("as well as any local..."). Is it like "as well as <iterate over> any local files"?

Reply:
yes exactly, let me rephrase